110. Balanced Binary Tree.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Definition for a binary tree node.
+ * public class TreeNode {
+ *     int val;
+ *     TreeNode left;
+ *     TreeNode right;
+ *     TreeNode(int x) { val = x; }
+ * }
+ */
+public class Solution {
+    public boolean isBalanced(TreeNode root) {
+        boolean output = true;
+        if (root == null) {
+            return true;
+        }
+        
+        if (isBalanced(root.left) == false || isBalanced(root.right) ==false ){
```

Comment:
lets merge line 17 and 21, since they both return false. Also add a space to the `=` (`==false`)